src/chart/pie/labelLayout.js

```diff
@@ -222,7 +222,7 @@ export default function (seriesModel, r, viewWidth, viewHeight, sum) {
         var font = labelModel.getFont();
 
         var labelRotate = labelModel.get('rotate')
-            ? (dx < 0 ? -midAngle + Math.PI : -midAngle) : 0;
+            ? +labelModel.get('rotate') * (Math.PI / 180) : 0;
```

Comment:
If `rotate` is not set, `midAngle` should be used.

Reply:
Note that you should also update ECharts doc at [here (cn)](https://github.com/ecomfe/echarts-doc/blob/gh-pages/cn/option/series/pie.md) and [here (en)](https://github.com/ecomfe/echarts-doc/blob/gh-pages/en/option/series/pie.md).

Reply:
midAngle is added, and I will update doc